doc/conf_ref.rst

```diff
@@ -458,7 +458,7 @@ configuration file by your distribution to override the DNF defaults.
 ``enabled``
     :ref:`boolean <boolean-label>`
 
-    Include this repository as a package source. The default is True.
+    Include this repository as a package source. The default is True. A repository that does not have the `enabled=0` entry is enabled by default.
```

Comment:
I thing that additional sentence is more confusing that the original statement.
The sentence `The default is True.` described behaviour in case that enable value is not specify. Additional information mentioned `enabled=0`, but what about 'enabled=No'.